channels/serializers.py

```diff
@@ -233,6 +247,21 @@ def get_upvoted(self, instance):
         """Is a comment upvoted?"""
         return instance.likes is True
 
+    def get_profile_image(self, instance):
+        """Find the Profile for the comment author"""
+        # context["profile_images"] is set only when doing GET
+        # on comment list view. else we're dealing with one record,
+        # so we can get the profile image from instance.author
+        if self.context.get("profile_images"):
```

Comment:
Can you change this to `if "profile_images" in self.context`?

Reply:
ah, didn't know I could do that! snazzy